src/__tests__/__browser__/client.js

```diff
@@ -21,8 +21,9 @@ test('client side app', async t => {
   const root = setup();
 
   const app = new App(React.createElement('span', null, 'hello'));
+  const simulator = getSimulator(app);
   try {
-    const ctx = await run(app, '/');
+    const ctx = await simulator.run(app, '/');
```

Comment:
Did this run correctly (instead of using `.render`)?

Reply:
Not sure if I see the output in Buildkite, maybe we got rid of tests running in folders in the latest CUP upgrade? This file was moved into src/__tests__/client.browser.js, so let's see what happens after rebase.